addOns/automation/src/main/javahelp/org/zaproxy/addon/automation/resources/help/contents/environment.html

```diff
@@ -93,7 +94,30 @@ <h3>Variables</h3>
 Variables can be defined in the 'vars' section.
 They can be hardcoded, refer to other variables or refer to system environment variables in the same way as above, e.g. '${envvar}'. 
 In case there are two variables with the same name, the value of the system variable would be preferred. 
-  
+
+<h3>Configs</h3>
+
+The 'configs' section can be used to define any value that can be set in the ZAP configuration file, which works in the same way as the ZAP '-config' command line parameter.
+<p>
+The advantages of using this section instead of the command line are:
+<ol>
+<li>The configuration will be self contained in the plan
+<li>The ZAP team will monitor they keys used in this way (when telemetry is enabled) and prioritise adding full support for the most frequently used one
```

Comment:
"options" vs "one"....just "one" suggests that we're literally 'expanding' to accommodate a single new option 😀